_posts/en/newsletters/2020-08-12-newsletter.md

```diff
@@ -0,0 +1,206 @@
+  within the context of eltoo.
+
+    The advantage of eltoo over the currently used LN-penalty mechanism
+    is that the publication of old eltoo channel states onchain doesn't
+    prevent the publication of the ultimate channel state.  This is
+    achieved in eltoo by creating signatures using `SIGHASH_ANYPREVOUT`
+    so that the signature from the ultimate state can spend bitcoins
+    from either the initial state, the penultimate state, or any state
+    in between.  However, transactions still need to identify which
+    state (previous output) they are attempting to spend.
+
+    One problem with that mechanism is that both an attacker and an
+    honest user could both try spending from the same previous state.
+    Miners and relay nodes would only keep one of those transactions in
+    their mempool, and there may be ways the attacker could ensure it
+    was an old state.  Another problem is that the attacker could
+    possibly trick the honest user into spending from a state that relay
+    nodes hadn't seen and so relay nodes would possibly reject the
+    unconfirmed transaction.  Although neither of these problems would
+    prevent the ultimate state from eventually being confirmed onchain,
+    they could be used for [transaction pinning][topic transaction
+    pinning] to prevent one of the honest user's time-sensitive
+    transactions from confirming in time.  These problems are similar to
+    the attack against LN payment atomicity described in [Newsletter
+    #95][news95 ln atomicity].  One proposed mitigation would be to have
+    special nodes (perhaps part of LN routing software) that looked for
+    cases where eltoo was being used and could use their knowledge of
+    that protocol to tell miners which transaction would truly be most
+    profitable to mine.
+
+## Field report: How segwit and batching could have saved half a billion dollars in fees
+
+{% include articles/veriphi-segwit-batching.md %}
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Implement ADDRv2 support (part of BIP155)][review club #19031] is a PR
+([#19031][Bitcoin Core #19031]) by Vasil Dimov that proposes a partial
+implementation of the [BIP155][] `addrv2` P2P message.
+
+The discussion covered the current `addr` message in Bitcoin Core, the BIP155
+`addrv2` message, how to signal support for `addrv2` on the P2P network, and the
+necessity of `addrv2` for the Bitcoin network to upgrade to Tor v3. Tor v2
+deprecation begins mid-September!
+
+{% include functions/details-list.md
+  q0="How does Bitcoin Core handle network addresses?"
+  a0="Bitcoin Core currently sees all peer addresses as being more or less in a
+      [single][19031 single] IPv6-like space. IPv4 and Tor v2 addresses are
```

Comment:
What is this syntax `[single][19031 single]`? Where can I see how it is rendered? https://github.com/bitcoinops/bitcoinops.github.io/blob/8c021652cd78cdf190a566cc02b41b107a8ec2d0/_posts/en/newsletters/2020-08-12-newsletter.md looks bad.

Reply:
(explained the link format via irc)

(you can render the document by building and running make preview per the readme and opening localhost:4000)